mixnet/src/crypto.rs

```diff
@@ -40,6 +40,26 @@ impl Default for PrivateKey {
     }
 }
 
+impl Serialize for PrivateKey {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        SerializablePrivateKey::from(self).serialize(serializer)
+    }
+}
```

Comment:
Why do we want to serialize a private key? Deserialization is ok to be read from config or whatever. But, as part of the protocol, do we need to send it somewhere? Otherwise probably wiser to not having the possibility of sending them.

Reply:
I just actually removed this file `crypto.rs`. As we discussed in https://github.com/logos-co/nomos-node/pull/591#discussion_r1510692858, we can accept `[u8; 32]` and convert it directly into `sphinx_packet::crypto::PrivateKey` when needed. So, we don't need the wrapper `crypto.rs`.